docs/installation/Installing_ACME_Responder.md

```diff
@@ -0,0 +1,186 @@
+Installing ACME Responder
+=========================
+
+## Overview
+
+This document describes the process to install an ACME responder on a PKI server that already has a CA subsystem.
+It assumes that the CA was [installed](Installing_CA.md) with the default instance name (i.e. pki-tomcat).
+
+**Note:** The ACME responder is currently a tech preview which means:
+* It is not intended for production.
+* It may corrupt your data.
+* There is no guarantee for correctness, security, or performance.
+* There is no guarantee for documentation or support.
+* The API, configuration, or the database may change in the future.
+* There may be no easy upgrade path to the future version. 
+
+## Installing SANToCNDefault Policy
+
+The SANToCNDefault is a certificate profile policy which generates
+a default subject DN for the certificate in case the CSR does not provide one.
+The subject DN will be generated from the first DNS name in the the SAN extension.
+
+This policy is needed by the ACME profile, but currently it is not installed by default in the CA,
+so it has to be added manually.
+
+To add the policy, edit /etc/pki/pki-tomcat/ca/registry.cfg as follows:
+
+```
+defaultPolicy.ids=...,sanToCNDefaultImpl
+defaultPolicy.sanToCNDefaultImpl.class=com.netscape.cms.profile.def.SANToCNDefault
+defaultPolicy.sanToCNDefaultImpl.desc=SAN to CN Default
+defaultPolicy.sanToCNDefaultImpl.name=SAN to CN Default
+```
+
+**Note:** Restart the server to enable the policy.
+
+## Installing ACME Profile
+
+The acmeServerCert.cfg is a sample profile for generating server certificates via ACME responder.
+
+This profile is currently not installed by default in the CA, so it needs to be added and enabled manually.
+
+To add the profile, execute the following command:
+
+```
+$ pki -u caadmin -w Secret.123 ca-profile-add /usr/share/pki/ca/profiles/acmeServerCert.cfg --raw
+```
+
+To enable the profile, execute the following command:
+
+```
+$ pki -u caadmin -w Secret.123 ca-profile-enable acmeServerCert
+```
+
+## Installing ACME Responder
+
+To install the ACME responder on PKI server, execute the following command:
+
+```
+$ pki-server acme-create
+```
+
+The command will create the initial configuration files in /etc/pki/pki-tomcat/acme folder.
+
+## Configuring ACME Responder Database
+
+The database configuration for the ACME responder is located at /etc/pki/pki-tomcat/acme/database.json.
+
+To use an in-memory database, copy the sample configuration file with the following command:
+
+```
+$ cp /usr/share/pki/acme/conf/database/in-memory/database.json \
+    /etc/pki/pki-tomcat/acme/database.json
+```
+
+Alternatively, edit the file as follows:
```

Comment:
nit: Alternatively, _create_ the file as follows.

Reply:
The file is actually already created earlier by the `pki-server acme-create` command. How about _configure_ instead of _create_?

Reply:
Ah, my understanding was bad. You can leave this alone if you'd prefer _edit_.

Reply:
OK, _edit_ is probably fine for now. We can always update it later if necessary.